_dropin-templates/dropin-installation.mdx

```diff
@@ -2,24 +2,24 @@
 title: DROPIN_NAME installation
 description: Learn how to install the DROPIN_NAME drop-in component into your site.
 sidebar:
-  # label: DROPIN_NAME Installation
+  label: Installation
   order: 2
 ---
 
-import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
 import Tasks from '@components/Tasks.astro';
 import Task from '@components/Task.astro';
 import Callouts from '@components/Callouts.astro';
 import Diagram from '@components/Diagram.astro';
 import Vocabulary from '@components/Vocabulary.astro';
 
-The DROPIN_NAME drop-in component, like our other drop-in components, are designed for the browser's JavaScript runtime without the need for a bundler. You can also install and execute drop-in components in a build-time environment with bundlers like Vite. The installation steps for both runtime and build-time environments are the same after the initial drop-in component package imports.
+{/* DESCRIBE the DROPIN_NAME drop-in component installation requirements and environment compatibility. Include information about runtime vs build-time environments and bundler support. */}
```

Comment:
We don't have use cases for "build-time" environments, nor bundler support in the Commerce Boilerplate. We should base our installation docs on the runtime (browser), which is what we follow in the Commerce Boilerplate.